test/Dapr.Client.Test/TypeConvertersTest.cs

```diff
@@ -38,11 +20,13 @@ public void AnyConversion_JSON_Serialization_Deserialization()
                 Name = "test"
             };
 
-            var any = TypeConverters.ToAny(response);
-            var type = TypeConverters.FromAny<Response>(any);
+            var options = new JsonSerializerOptions(JsonSerializerDefaults.Web);
+
+            var any = TypeConverters.ToJsonAny(response, options);
+            var type = TypeConverters.FromJsonAny<Response>(any, options);
 
             type.Should().BeEquivalentTo(response);
-            any.TypeUrl.Should().Be("Dapr.Client.Test.TypeConvertersTest+Response");
+            any.TypeUrl.Should().Be(string.Empty);
```

Comment:
note, this property sets itself to empty string. I didn't make it do that, it does it on its own.